### PR TITLE
feature(lib): added graphql mutation for updating a promo

### PR DESCRIPTION
**Pull request by lafbelmonte**
-added graphql mutation functionality to update a promo

### CODE REVIEW

File: koa-crud/src/use-cases/promos/update-promo.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      _id: id,
+    });
+
+    if (!promoExists) {
+      throw new Error(`Promo ID doesn't exist`);
+    }
+
+    const promo = await promoEntity(info);
+
+    const truthlyPromo = await R.filter(R.complement(R.isNil))(promo);
```
Comment (ralphcasipe1) [inline koa-crud/src/use-cases/promos/update-promo.ts]: you can refactor this to: 
```ts
R.filter(Boolean)(promo)
```

Comment (ralphcasipe1) [reply koa-crud/src/use-cases/promos/update-promo.ts]: @lafbelmonte  Do we still need to `await` here?

**Review by ralphcasipe1 — CHANGES_REQUESTED** [review]
@lafbelmonte  Please see my small requested changes.

File: koa-crud/src/use-cases/promos/update-promo.ts (hunk 1 above)
Comment (lafbelmonte) [reply koa-crud/src/use-cases/promos/update-promo.ts]: No sir, no promise inside the function, will remove the await.

**Review by ralphcasipe1 — APPROVED** [review]
:tada:  LGTM!

**Pull request merged**